scripts/zones/Empyreal_Paradox/mobs/Promathia_2.lua

```diff
@@ -67,7 +67,7 @@ entity.onSpellPrecast = function(mob, spell)
     end
 end
 
-entity.onMagicCastingCheck = function(mob, target, spell)
+entity.onMobMagicPrepare = function(mob, target, spell)
     if math.random() > 0.75 then
         return 219
```

Comment:
I know its comment and meteor from memory, but lets comment them for ppl who won't know what the numbers are

Reply:
Sure this was just an un-related fix that you and I had talked about but i'll add the 'what this is' to the comments

Reply:
ty